examples/regression/4-tutorials/plot_cqr_tutorial.py

```diff
@@ -197,14 +198,14 @@ def plot_prediction_intervals(
     axs.errorbar(
         y_test_sorted_[~warnings],
         y_pred_sorted_[~warnings],
-        yerr=error[~warnings],
+        yerr=np.abs(error[~warnings]),
```

Comment:
Not really clear to me why we add `np.abs()`?

Reply:
Sometimes with CQR the error is negative and the errorbar plot is not possible with a negative value